### PR TITLE
Add string examples

### LINKED ISSUE #52: example: strings

### PR DESCRIPTION
**Pull request by spmse**
added examples for strings.
Possibly fixes #52

### CODE REVIEW

File: examples/strings/strings.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+println('age = $age')           // age = 25
+```
+
+To define character literals use: ` `` `. Raw strings can be defined prepending `r`. They are not escaped.
+
+```go
+hello := 'Hello\nWorld'
+println(hello)                  // Hello
+                                // World
+raw_hello := r'Hello\nWorld'
```
Comment (spmse) [inline examples/strings/strings.md:90]: This causes error of TypoCI but it is correct. Can I somehow ignore TypeCI?

Comment (dhonx) [reply examples/strings/strings.md:90]: @SuicideS3ason add the ignored words to [this file](https://github.com/v-community/v_by_example/blob/master/.typo-ci.yml#L11)

File: examples/strings/strings.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Strings
+ 
+In V one can define strings using the `:=` operator. Strings (like other variables) are immutable by default. One is free to use `""` or `''` to denote a string. When using `vfmt` all double quoted strings will be converted to single quoted ones unless it contains a single quote character.
```
Comment (dhonx) [inline examples/strings/strings.md]: I think `double-quoted` is better

File: examples/strings/strings.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+We therefore need to convert it to string by using `.str()` or use string interpolation (preferred):
+
+```go
+age := 25
+println('age = ' + age.str())   // age = 25
+println('age = $age')           // age = 25
+```
+
+To define character literals use: ` `` `. Raw strings can be defined prepending `r`. They are not escaped.
```
Comment (dhonx) [inline examples/strings/strings.md]: add `as` after `defined`

**Review by dhonx — CHANGES_REQUESTED** [review]
Great work !!!  👌

File: examples/strings/strings.md (hunk 2 above)
Comment (spmse) [reply examples/strings/strings.md]: added in 4a7570c

File: examples/strings/strings.md (hunk 3 above)
Comment (spmse) [reply examples/strings/strings.md]: added in 4a7570c

**Review by dhonx — APPROVED** [review]
Great !! Ready to merge  👌

Thanks @SuicideS3ason

**Pull request merged**